### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by Flavio0834**
### Description
Started creating endpoints for users and associations.
Endpoints linked to empty async functions.

### CODE REVIEW

File: .github/workflows/lintandtest.yml [hunk 1]
```diff
@@ -30,10 +30,5 @@ jobs:
       - name: Format with black
         uses: psf/black@stable
 
-      - name: Sort imports with isort
-        uses: jamescurtin/isort-action@master
-        with:
-          requirementsFiles: "requirements.txt"
-
```
Comment (AtomHare) [inline .github/workflows/lintandtest.yml:37]: We should keep this

File: app/endpoints/amap.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/products")
```
Comment (AtomHare) [inline app/endpoints/amap.py:6]: we need to have `amap` on all those requests as the endpoint
```suggestion
@router.get("/amap/products")
```

File: app/endpoints/bdecalendar.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/events")
```
Comment (AtomHare) [inline app/endpoints/bdecalendar.py:6]:
```suggestion
@router.get("/calendar/events")
```
like amap

**Review by AtomHare — CHANGES_REQUESTED** [review]
Pretty good, 
You just need to edit the routes and I'm not sure about the naming of `bdebooking.py` and `bdecalendar.py`

File: .github/workflows/lintandtest.yml (hunk 1 above)
Comment (Flavio0834) [reply .github/workflows/lintandtest.yml:37]: done

File: app/endpoints/amap.py (hunk 2 above)
Comment (Flavio0834) [reply app/endpoints/amap.py:6]: Adding prefix in include_router in api.py is better but same effect. Done

File: app/endpoints/bdecalendar.py (hunk 3 above)
Comment (Flavio0834) [reply app/endpoints/bdecalendar.py:6]: same answer

**Review by Victor-Angot — APPROVED** [review]

**Review by Jhobahtes — APPROVED** [review]
LGTM 👍

**Review by AtomHare — APPROVED** [review]
LGTM

**Pull request merged**